napari/_qt/qt_dims.py

```diff
@@ -147,6 +158,8 @@ def _create_sliders(self, number_of_sliders):
             slider = self._create_range_slider_widget(dim_axis)
             self.layout().addWidget(slider)
             self.sliders.insert(0, slider)
+            self._slider_axis = list(np.add(self._slider_axis, 1))
```

Comment:
maybe want to use a list comprehension instead but i think it's fine

Reply:
fixed